charts/seed-operatingsystemconfig/original/templates/kubelet/_kubelet.flags

```diff
@@ -15,6 +15,9 @@
 {{- if semverCompare "< 1.14" .Values.kubernetes.version }}
 --feature-gates=SupportPodPidsLimit=true
 {{- end }}
+{{- if .Values.enableContainerD }}
```

Comment:
Can you add the new Helm chart values to the `values.yaml` so that one can see which values are possible?

Reply:
Do you mean adding the values:
--container-runtime=remote
--container-runtime-endpoint=/run/containerd/containerd.sock
?

Reply:
No, I mean the `enableContainerD`, for example. It's a new value that you added for the Helm chart, and usually we document all possible values to the `values.yaml`.

Reply:
Ok i got you,
I will add it.